src/libraries/System.IO.Compression/src/System/IO/Compression/ZipBlocks.cs

```diff
@@ -534,6 +552,24 @@ public static bool TryValidateBlock(BinaryReader reader, ZipArchiveEntry entry)
 
             return true;
         }
+
+        internal static ulong ConvertToUlong(uint loverAddressValue, uint higherAddressValue)
```

Comment:
this can be `private`

Reply:
`loverAddressValue` => `lowerAddressValue`

Reply:
> loverAddressValue => lowerAddressValue

Thas is from WPF 😛